docs/event-dispatcher.md

```diff
@@ -67,10 +67,39 @@ Example:
       "tx_index": 1,
       "txid": "0x738e4d44636023efa08374033428e44eca490582bd39a6e61f3b6cf749b4214c"
     }
-  ]
+   ],
+   "matured_miner_rewards": [
```

Comment:
Should this include the associated coinbase txid? Thinking about this will be implemented in the API db

Reply:
Hmmm -- these matured rewards don't have an exact correspondence with coinbase transactions -- a miner may be receiving matured rewards for microblocks that are being confirmed by a subsequent miner, microblocks they confirmed, etc. This field could add the reward's block hash, which would let the API correlate the rewards with the mined block that generated those rewards.

Reply:
> This field could add the reward's block hash

Makes sense. Should the reward's block hash be used in the API's re-org handling?

Reply:
I don't think so. I'll add two fields:

```
from_index_block_hash
from_stacks_block_hash
```

The index block hash will let you pair any rewards to the re-org'ed block.